apps/coordinator/src/components/Wallet/WalletSpend.jsx

```diff
@@ -62,6 +64,12 @@ class WalletSpend extends React.Component {
     }
   };
 
+  handleFeeEstimate = (feeEstimate) => {
+    const wasteMetrics = new WasteMetrics();
+    const dustLimits = wasteMetrics.calculateDustLimits(feeEstimate);
```

Comment:
Based on the description in the code and readme, I would've expected "spend waste metric" to be useful here:

```
     Spend Waste Score (S.W.S)
    Definition : 
      A score that indicates whether it is economical to spend a particular output now 
      or wait to consolidate it later when fees could be low.
```

Reply:
This was more about the transaction which we are forming and not for single UTXO. As commented here https://github.com/caravan-bitcoin/caravan/pull/112#discussion_r1726336091

We can manually select more than one UTXOs to make a transactions so I was not sure if this could be the correct metric to tell if a single UTXO should be present in wallet or should be spent or consolidated etc. This could fit more into the preview transaction tab where we have all the details right in place. And there we can suggest, If making this transaction now is going to save some sats or is going to affect badly in long term.

Reply:
I see! Yeah that makes sense. Thanks for the explanation!